bin/runtime/src/lib.rs

```diff
@@ -60,6 +60,10 @@ use sp_runtime::traits::One;
 pub use sp_runtime::BuildStorage;
 pub use sp_runtime::{FixedPointNumber, Perbill, Permill};
 
+mod bag_thresholds;
+mod custom_migrations;
+mod weights;
+
```

Comment:
You forgot to bump `spec_version`.

Reply:
And it's better to bump `transaction_version` as well, as the docs say to do that when a new pallet enters.

Reply:
Done.